packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -786,6 +788,21 @@ const Component = CoreView.extend(
       );
     },
 
+    on(eventName: string) {
+      let owner = getOwner(this);
+      if (owner.lookup<EmberVMEnvironment>('-environment:main')!.isInteractive) {
+        let eventDispatcher = owner.lookup<EventDispatcher>('event_dispatcher:main');
+        if (
+          eventDispatcher &&
+          Array.from(eventDispatcher.lazyEvents.values()).includes(eventName)
```

Comment:
I don't think we can use Array.from or includes due to IE11 issues

Reply:
Yeah, already noticed that it breaks the Browserstack CI job. Will fix it later today...

Also `.values()` btw